GrandCentralBoard/Widgets/Harvest/View/AreaBarChartViewModel.swift

```diff
@@ -20,21 +20,31 @@ struct AreaBarItemViewModel {
     let valueLabelMode: AreaBarItemValueLabelDisplayMode
 }
 
-struct AreaBarChartViewModel {
+struct AreaBarChartComponentViewModel {
     let barItems: [AreaBarItemViewModel]
+    let horizontalAxisCountLabelText: String
+    let headerText: String?
+    let subheaderText: String?
+}
+
+struct AreaBarChartViewModel {
+
+    let mainChart: AreaBarChartComponentViewModel
+    let componentCharts: [AreaBarChartComponentViewModel]
+
     let horizontalAxisStops: Int
     let horizontalAxisLabelText: String
-    let hotizontalAxisCountLabelText: String
+
     let centerText: String?
-    let headerText: String?
-    let subheaderText: String?
 
-    static func emptyViewModel(header header: String? = nil, subheader: String? = nil) -> AreaBarChartViewModel {
+    static func emptyViewModel(header header: String? = nil, subheader: String? = nil, historicalCharts: [AreaBarChartComponentViewModel] = []) -> AreaBarChartViewModel {
```

Comment:
What if a historical chart is empty? Shouldn't we have a AreaBarChartComponentViewModel.emptyViewModel method as well, and use it inside this method?